app/views/adoption_application_mailer/_adoption_application_mailer_content.html.erb

```diff
@@ -0,0 +1,50 @@
+<table style="border-collapse: collapse;">
+  <%= render "table_row", adoption_application: adoption_application,
```

Comment:
Not sure how I feel about this partial usage. It's still a lot of repetition. You could do something like.

```ruby
class AdoptionApplication
  PUBLIC_FIELDS =  %i(name address city cell state zip email current_rates
      current_rat_count current_rat_gender current_rat_age previous rates
      previous_rate_count previous_rat_lifespan cause_of_death vet_care
      vet_name  cage_description cage_location household_members
      children_ages additional_animals interactions_time preferred_characteristics)
```

then here maybe

```erb
<% AdoptionApplication::PUBLIC_FIELDS.each do |field| %>
  <tr>
    <td style="border:solid; border-width:1px; font-weight:bold;"><%= field %></td>
    <td style="border:solid; border-width:1px; width: auto;"><%= @adoption_application.attributes[field] %></td>
  </tr>
<% end %>
```